symplyphysics/laws/astronomy/maximum_angle_of_rotation_during_gravitational_maneuver.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve, atan)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+
+# Description
+## A gravitational maneuver is a purposeful change in the trajectory and flight speed of a spacecraft under the influence
+## of the gravitational fields of celestial bodies.
+## The maximum angle of rotation of the rocket during a gravitational maneuver depends on the first cosmic velocity of
+## the planet around which the maneuver is performed, and the speed of the rocket relative to this planet.
```

Comment:
From your description someone may think that there are various angles during gravitiational maneuver. And you should choose any until it reaches maximum.
But actually this law describes the closest possible maneuver, when aiming range is at planet radius. Add it do description and to conditions.